template-reactless/my_component/frontend/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "streamlit_component_template",
+  "name": "my_component",
```

Comment:
Again, this should be either `my_component` or we should replace `my_component` everywhere with proper name

Reply:
This is a rendered template, so we should have a specific name here.

Reply:
Should not this replaced with `{{ cookiecutter.import_name }}` ?